samples/ccr/stop.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+#source .elastic-version
```

Comment:
I think this line can be removed.

Similar to the comment on `start.sh` it would be nice if there was some kind of comment describing the usage for this file

Reply:
Added the comment.